### PR TITLE
Test script for issue 3859

### PR DESCRIPTION
**Pull request by GetmanetsIrina**
ATF Test Scripts to check [FORDTCN-6937](https://luxproject.luxoft.com/jira/browse/FORDTCN-6937)

This PR is **ready** for review.

### Summary
Added test script for 3859 issue

### ATF version
develop

### Changelog
- new script added
- test set for defects 8_1 has been updated with the new script

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_sets/Defects/Defects_release_8_1.txt [hunk 1]
```diff
@@ -20,3 +20,4 @@
 ./test_scripts/Defects/8_1/3845/3845_4_PerformAudioPassThru_custom_success_codes_in_error_structure_to_ui.lua
 ./test_scripts/Defects/8_1/3845/3845_5_PerformAudioPassThru_custom_success_codes_in_error_structure_to_tts.lua
 ./test_scripts/Defects/8_1/3845/3845_6_PerformAudioPassThru_custom_success_codes_in_error_structure_to_tts_and_ui.lua
+./test_scripts/Defects/8_1/3859_CreateInteractionChoiceSet_infor_in_response_during_erroneous_response.lua
```
Comment (VjKlepikov) [inline test_sets/Defects/Defects_release_8_1.txt:23]: I propose to rename infor -> info according to parameter name

File: test_scripts/Defects/8_1/3859_CreateInteractionChoiceSet_infor_in_response_during_erroneous_response.lua [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+-- Description: Check that SDL transfers the error message from the HMI response to the mobile app
+
+-- Steps:
+-- 1. HMI and SDL are started
+-- 2. Mobile app is registered and activated
+-- 3. Mobile app requests CreateInteractionChoiceSet RPC
+-- 4. SDL sends VR.AddCommand(type = "Choice") request to the HMI
+-- 5. HMI responds with erroneous code and error message to the VR.AddCommand(type = "Choice") request
+-- SDL does:
+-- 1. process the CreateInteractionChoiceSet response from HMI
```
Comment (VjKlepikov) [inline test_scripts/Defects/8_1/3859_CreateInteractionChoiceSet_infor_in_response_during_erroneous_response.lua:13]:
```suggestion
-- 1. process the VR.AddCommand response from HMI
```

File: test_scripts/Defects/8_1/3859_CreateInteractionChoiceSet_infor_in_response_during_erroneous_response.lua [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  "ABORTED",
+  "IGNORED",
+  "IN_USE",
+  "TIMED_OUT",
+  "INVALID_DATA",
+  "CHAR_LIMIT_EXCEEDED",
+  "INVALID_ID",
+  "DUPLICATE_NAME",
+  "GENERIC_ERROR"
+}
```
Comment (VjKlepikov) [inline test_scripts/Defects/8_1/3859_CreateInteractionChoiceSet_infor_in_response_during_erroneous_response.lua:49]: I propose to extend coverage for errorcode:
 "UNSUPPORTED_REQUEST",
  "USER_DISALLOWED",
  "READ_ONLY"

**Comment by GetmanetsIrina** [comment]
@VjKlepikov done in [19c58d0](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/162/commits/19c58d0e05688645565bc7269d60642e3223f442)

**Review by VjKlepikov — APPROVED** [review]

**Comment by GetmanetsIrina** [comment]
moved to open

**Pull request closed without merging**